scripter/scripter.go

```diff
@@ -175,3 +172,76 @@ func ReloadScripts(s Scripter) {
 		}
 	}
 }
+
```

Comment:
Can you place comments or split this into subfunctions?

Reply:
Yes!

Reply:
Done!